src/components/Filter.tsx

```diff
@@ -1,34 +1,42 @@
-import React, { ReactElement } from 'react';
-import { Form, FormGroup, Label, Input } from 'reactstrap';
+import React from 'react';
+import Select from 'react-select';
+import _ from 'lodash';
+import { ValueType } from 'react-select/src/types';
 
-const Filters = ({ layers, setLayers }) => {
-  const y = [];
-  Object.keys(layers).map(function (key, index) {
-    y.push(
-      <FormGroup check>
-        <Label check>
-          <Input
-            type="checkbox"
-            checked={layers[key]}
-            name={key}
-            onChange={handleLayer}
-          />{' '}
-          {key}
-        </Label>
-      </FormGroup>,
-    );
-  });
+const Filters = ({
+  layers,
+  setLayers,
+}: {
+  layers: { [k: string]: boolean };
+  setLayers: (val: { [k: string]: boolean }) => void;
+}) => {
+  const selectOptions = _.map(layers, (layer, key) => ({
+    value: key,
+    label: key,
+  }));
 
-  function handleLayer(event: { target: { name: string; checked: boolean } }) {
-    const keyx = event.target.name;
-    const newLayers = { ...layers };
-    newLayers[keyx] = event.target.checked;
+  function handleLayer(
+    event: Array<{ value: string; label: string }> | null | undefined,
+  ) {
+    if (Array.isArray(event) && event.length === 0) event = null;
+    const selectedLayers = _.map(event, (layer) => layer.label);
+    const newLayers: { [x: string]: boolean } = {};
+    _.forEach(selectOptions, (option) => {
+      newLayers[option.label] = event
+        ? _.includes(selectedLayers, option.label)
+        : true;
+    });
     setLayers(newLayers);
   }
 
   return (
-    <div id="filters">
-      <Form>{y}</Form>
+    <div className="filter">
+      <Select
```

Comment:
If we could change the text inside the box to say 'Filter...' that would be great!